bool activate
---
bool is_valid
float32 bestpos
